unit_tests/test_argus_results.py

```diff
@@ -0,0 +1,8 @@
+from sdcm.sct_events.event_counter import STALL_INTERVALS
+from sdcm.utils.decorators import ReactorStallStatsResult
+
+
+def test_reactor_stalls_intervals():
+    """protecting against changes in the intervals so result table columns are always correct"""
```

Comment:
why not code it, that it add the columns base on the same definition ?

Reply:
coded, test replaced with one that tests whole sending process.